README.md

```diff
@@ -4,22 +4,98 @@
   <a href="https://badge.fury.io/py/simexpal"><img src="https://badge.fury.io/py/simexpal.svg"></a>
 </p>
 
-## 
-simexpal is a collection of tools to manage, launch, monitor and evaluate algorithmic experiments
-(e.g. performance benchmarks).
-The goal of this toolbox is to automatize various repetitive tasks that occur whenever such experiments
-need to be executed. simexpal consists of both a command line (CLI) utility and a Python package.
-While the CLI utility can be used to perform many common tasks with minimal configuration,
-the Python package can be employed when more extensive customization is necessary.
+# simexpal 
 
-**Features.** simexpal assists users with the following operations
-- Manage instances: **Download instances** from external sources, **collect information** about instances.
-- Manage runs: **Launch** runs and **monitor** their progress, **detect failed runs** and **restart** them, **collect results** of runs.
+_simexpal_ is a python based tool to setup, manage, launch, monitor, and
```

Comment:
"python" should probably be uppercase here.

Reply:
Thanks :heavy_check_mark: